src/vnsw/opencontrail-vrouter-netns/opencontrail_vrouter_netns/vrouter_hyperv.py

```diff
@@ -14,7 +14,7 @@
 from common import validate_uuid
 
 
-def powershell(cmds):
+def call_powershell(cmds):
```

Comment:
I don't like this function. It is just kind of wrapper and it is *only* adding "powershell.exe" to cmds. You have to still add a lot of arguments when you want to invoke some command. The only adantage of this funtion is that you don't have to add three arguments to every invocation of subprocess. Also, there is no any error checking. 

This kind of design can create a lot of places where you can make mistake, or/and debugging this code is very hard. The solution maybe is more 'pythonic' but I think that it can create a lot of problems in future. 

I think that you should add the class which will be middleware between subprocess and your code. For every command which you want to execute from powershell. If you do something like that, then you can add more checks to verify your command (and as we now, we don't create simple few arguments command, but our commands are more complicated). I think that verifing how exactly look every command (and logging this to file) is very important for improving time of solving debugging problem.